examples/nidaqmx_analog_input/measurement.py

```diff
@@ -9,19 +9,13 @@
 
 import ni_measurementlink_service as nims
 
-measurement_info = nims.MeasurementInfo(
-    display_name="NI-DAQmx Analog Input (Py)",
+parent_directory = pathlib.Path(__file__).resolve().parent
```

Comment:
I think I'd prefer a name like 'root_directory' or 'service_directory'. What do you think?